cloudflared/rootfs/etc/s6-overlay/s6-rc.d/cloudflared/run

```diff
@@ -3,23 +3,39 @@
 # Home Assistant Add-on: Cloudflared
 # Runs the Cloudflare Tunnel for Home Assistant
 # ==============================================================================
-declare default_config=/tmp/config.json
+declare config_file="/tmp/config.json"
+declare certificate="/data/cert.pem"
+declare -a cloudflared_options
```

Comment:
This is great. Not sure though if we need the separation of "cloudflared_options" and "tunnel_options".

Looking at the [documentation](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/local-management/arguments/), you can add the "--no-autoupdate" also after "tunnel". I thin the same should be valid for the --post-quantum.

What do you think?

Reply:
You are right, I have changed it. That simplifies the whole thing enormously. 😉